### ISSUE TITLE
open to using a sass compiler for the styles?

### ISSUE THREAD

**Issue by mreinstein** [paypal/accessible-html5-video-player#126]
@hozefaj wondering if y'all would be open to moving the css  to https://sass-lang.com

Would be happy to send a PR with this if it would be welcome. Definitely not looking to override library/design decisions, so no biggie if this really isn't wanted. I do think it could be a nice addition and cut down on some of the repeated boilerplate in the `.css` though.

**Comment by weboverhauls** [comment]
The original intent of the project is to use "vanilla" JS and CSS so anyone can use it.

**Comment by mreinstein** [comment]
@weboverhauls right, I think this would still be the case if we adopted scss. We would generate something like `dist/px-video.css` which is what people would pull into their projects. The `.scss` file would only be relevant to people modifying this library and submitting their changes.

**Comment by shanepoints** [comment]
+1
This would also enable more flexibility for developers who pull this into their own projects to customize things like accent color or fonts.

**Comment by hozefaj** [comment]
Even with `scss` or `less` you anyone can directly pull the CSS file and update the styles as needed. 

What happens that there are other CSS preprocessors also that will get left out if we do `scss`. So agree with @weboverhauls comments to keep only CSS and folks should be able to import the CSS into any build process and update as needed. 

The idea is to have the code without any dependencies.

**Comment by shanepoints** [comment]
Hmmm.  That's a good point.  

But to the point of @mreinstein, there would still be a vanilla css file output from the processed Sass, so it would still plug and play that way. It would just add one more option to people that _do_ use Sass, which is the [majority of the pre-process people](https://www.keycdn.com/blog/sass-vs-less).
